### PR TITLE
A bunch of fixes on translations area

### PR DESCRIPTION
**Pull request by Caballlero**
Done.

### CODE REVIEW

**Comment by Caballlero** [comment]
I think it ready now.

File: resources/qml/pages/LoginPage.qml [hunk 1]
```diff
@@ -13,7 +13,7 @@ Page {
         }
 
         PageHeader {
-            title: qsTr("Login")
+            title: qsTr("Wellcome to Shmoose")
```
Comment (geobra) [inline resources/qml/pages/LoginPage.qml]: You changed the title of the page. Not a bad idea. But you used a wrong spelling. Must be 'Welcome'.

**Comment by geobra** [comment]
Please use for the translation file name syntax like the other ones. 
de_DE or en_GB instead of harbour-shmoose-es.ts. Must be es_ES or something like that.

What is the additional harbour-shmoose.ts file about?

**Comment by Caballlero** [comment]
wellcome is for  uk-english
welcome is for usa-english
to be honest wellcome it much better that welcome. if you use google translate they are using usa-english.

**Comment by Caballlero** [comment]
harbour-shmoose.ts is the true file to translate at any idiom.

**Comment by d9h20f** [comment]
@Caballlero you are completely wrong. 'Welcome' is _always_ spelled with only one l. 'Wellcome' does not exist.

**Comment by d9h20f** [comment]
@geobra translation files will still work if they're called 'harbour-shmoose-es' instead of 'harbour-shmoose-es_ES'. The last part is only necessary if you need to explicitly specify that the translation can vary between regional varieties of a language, and is only taken into account if there are, in fact, multiple translations available in different varieties of the language.

**Comment by geobra** [comment]
You are right about the extension (es vs es_ES). But the current files are named de_DE.ts and en_GB.ts. In my opinion it makes sense to stay at one schema instead of mixing file names up. So please: 
- rename the harbour-shmoose-es.ts to es_ES.ts
- remove the file harbour-shmoose.ts. That makes no sense. If one would start another language, he can just grab any existing file and start over.
- resolve the conflict so that the pull request can be merged

**Comment by Caballlero** [comment]
Listo!

**Comment by geobra** [comment]
Looks very good now. Please just resolve the remaining conflict in the pro file and I will click the merge button into the project :-).

**Comment by Caballlero** [comment]
what conflict you still talk about?

**Pull request merged**